packages/hyperion-flowlet/src/FlowletWrappers.ts

```diff
@@ -178,7 +178,27 @@ export function initFlowletTrackers(flowletManager: FlowletManager) {
       }
       args[0] = flowletManager.wrap(handler, fi.name);
       return args;
-    })
+    });
+  }
+
+  for (const fi of [
+    IWindow.requestIdleCallback,
+  ]) {
+    fi.onBeforeCallMapperAdd(args => {
+      let handler = args[0];
+      args[0] = flowletManager.wrap(handler, fi.name);
+      return args;
+    });
+  }
+
+  for (const fi of [
```

Comment:
The implementation is the same, right? 

```suggestion
```

Reply:
It is the same, but the types where different enough that TS was complaining. Initially I put all of them in the same block as `setTimeout` and `setInterval`. I fiddle around a bit more to see if there is a way to make TS happy with the types.

Reply:
Nope, tried several things but could not make the type work.